front/app/containers/Admin/projects/components/AnalysisBanner.tsx

```diff
@@ -47,18 +58,27 @@ const AnalysisBanner = () => {
       clHistory.push(
         // TODO: Fix this the next time the file is edited.
```

Comment:
We are now trying to fix and remove the comment when we can for this type of check. I think it's relatively easy to do here.

Reply:
Guilty as charged!
I saw the comments, but was not 100% sure I could trust the typings, and was too lazy to go and check. Will do.